source/imaer-gml/src/test/java/nl/overheid/aerius/gml/GMLCalculationSetOptionsReaderTest.java

```diff
@@ -176,6 +178,25 @@ void testReadCalculationSetOptionsDuplicateOptions() {
     assertEquals("somewhere else", options.getNcaCalculationOptions().getPermitArea(), "PermitArea");
   }
 
+  @Test
+  void testOWN2000ReadCalculationSetOptions() {
+    final FeatureCollection featureCollection = mock(FeatureCollection.class);
+    final MetaData metaData = mock(MetaData.class);
+    final IsCalculationMetaData calculationMetaData = mock(IsCalculationMetaData.class);
+    final List<IsGmlProperty<IsCalculationOption>> suppliedOptions = new ArrayList<>();
+
+    suppliedOptions.add(mockCalculationOption("permit_lower_bound", "policy"));
+    when(featureCollection.getMetaData()).thenReturn(metaData);
+    when(calculationMetaData.getOptions()).thenAnswer(a -> suppliedOptions);
+    when(metaData.getCalculation()).thenReturn(calculationMetaData);
+    final GMLCalculationSetOptionsReader reader = new GMLCalculationSetOptionsReader(featureCollection);
+
+    final CalculationSetOptions options = reader.readCalculationSetOptions(Theme.OWN2000);
+    assertNotNull(options, "returned options shouldn't be null");
+    assertSame(PermitLowerBoundType.POLICY, options.getOwN2000CalculationOptions().getPermitLowerBoundType(),
```

Comment:
Doesn't matter too much, but generally we use `assertEquals`. Why `assertSame` in this case?